tests/common/load_generator.hpp

```diff
@@ -20,6 +22,10 @@ inline double_t constFunction(double_t x) {
   return 10;
```

Comment:
It is just '10'. Constant value. It should be named at least `const10Function` or need to make it parametrized.